Core/src/Seeding/EstimateTrackParamsFromSeed.cpp

```diff
@@ -39,6 +39,11 @@ Acts::FreeVector Acts::estimateTrackParamsFromSeed(const Vector3& sp0,
   Vector3 local1 = transform.inverse() * sp1;
   Vector3 local2 = transform.inverse() * sp2;
 
+  if (local2(1) == 0) {
```

Comment:
checking for `== 0` will not catch cases that are close to `0` but still fail

Reply:
The only failure cases I've seen were when this was exactly zero.